Austria/ruby/flag.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+# coding: utf-8
+
+require 'bundler/inline'
```

Comment:
am i wrong with the assumption that this would require bundler and colorize opposed to just requiring colorize?

Reply:
this requires `bundler/inline` which allows me to specify the Gemfile inline, which also automatically requires the gems specified in the `gemfile true do ... ` block

Reply:
and it also installs colorize if not already, so yay for that